WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -44,10 +44,22 @@
 static BOOL FBIncludeNonModalElements = NO;
 static NSString *FBAcceptAlertButtonSelector = @"";
 static NSString *FBDismissAlertButtonSelector = @"";
+static NSString *FBSnapshotMaxDepthKey = @"maxDepth";
+static NSMutableDictionary *FBSnapshotRequestParameters;
 
 
 @implementation FBConfiguration
 
++ (void)initialize
```

Comment:
Should it be `load`?

Reply:
Looks like more suitable is the ```initialize``` method for a class.
It is called just after ```load```, but before the class receives any message (method call).
It is guaranteed to be called once when a class is initialized. (It might be called multiple times, if a class has subclasses - when initializing each subclass, but the FBConfiguration is not the case). 

I've spent some time reading this doc, and then experimenting.
https://developer.apple.com/documentation/objectivec/nsobject/1418639-initialize?language=objc

feels like the ```load``` is more appropriate place for things like swizzling, and ```initialize``` - for initializing class variables.